client/src/components/DailyUserCalories/index.js

```diff
@@ -9,7 +9,7 @@ import Loading from '../Loading';
 
 const { bool } = PropTypes;
 
-function DailyUserCalories({ showCard, showChart }) {
+function DailyUserCalories({ showCard, showChart, displayNone, ...rest }) {
```

Comment:
Is `displayNone` not used in this component?  I don't see anywhere where you are actually using it in the code below.